packages/synapse-react-client/src/testutils/ComponentToBePassedPropsCustomMatcher.ts

```diff
@@ -21,10 +21,7 @@ expect.extend({
   },
   toHaveBeenRenderedWithProps(mockComponent: Mock, expectedProps: unknown) {
     try {
-      expect(mockComponent).toHaveBeenCalledWith(
-        expectedProps,
-        expect.anything(),
-      )
+      expect(mockComponent).toHaveBeenCalledWith(expectedProps, undefined)
```

Comment:
In React 19, a function component is called with `undefined` 2nd argument instead of an object.